packages/components/molecules/f-searchbox/src/components/FormStates/FormLoadingIndicator.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div
+        v-if="shouldDisplayLoadingIndicator"
+        data-test-id="loading-spinner-indicator"
+        :class="$style['c-spinner-wrapper']">
+        <div :class="$style['c-spinner']" />
+    </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+
+export default {
+    computed: {
+        ...mapState('searchbox', [
+            'isFullAddressSearchEnabled',
+            'isLoadingResults'
+        ]),
+
+        /**
+         * Only display loading spinner for Loqate feature for now.
+         *
+         * */
+        shouldDisplayLoadingIndicator () {
```

Comment:
Should this check better be in the parent component? And v-if statement on the component call?

Reply:
I did originally have it there but then im using this component in two places so you'd need the same logic (mapState & computed prop) in both places. Instead when it's here it's just in one place.